--- a/capnp/schema.capnp	2021-10-21 15:03:20.221774538 -0400
+++ b/capnp/schema.capnp	2021-10-21 15:04:20.870019327 -0400
@@ -82,7 +82,7 @@

     file @6 :Void;

-    struct :group {
+    struct :group $Go.name("structNode") {
       dataWordCount @7 :UInt16;
       # Size of the data section, in words.

@@ -336,7 +336,7 @@
       typeId @15 :Id;
       brand @21 :Brand;
     }
-    struct :group {
+    struct :group $Go.name("structType") {
       typeId @16 :Id;
       brand @22 :Brand;
     }
@@ -450,7 +450,7 @@
     list @14 :AnyPointer;

     enum @15 :UInt16;
-    struct @16 :AnyPointer;
+    struct @16 :AnyPointer $Go.name("structValue");

     interface @17 :Void;
     # The only interface value that can be represented statically is "null", whose methods always
